### ISSUE TITLE
Many developer tools

### ISSUE THREAD

**Issue by atlaschiew** [atlaschiew/laravel-study#5]
1) search code in all files
```sh
cd madxframework
grep -rnw './' -e 'search keywords'

# search only on php files
grep -rnw './' -e 'search keywords' --include=*.php
```

2) run composer
```sh
cd madxframework
/usr/local/bin/ea-php82 ~/composer.phar update
```

3. download composer.phar
```sh
# check latest composer.phar version in https://getcomposer.org/download/2.5.5/composer.phar
# then copy and run command below
cd ~
wget https://getcomposer.org/download/2.5.5/composer.phar
```

**Comment by atlaschiew** [comment]
In Laravel, `composer dump-autoload` is a command that updates the autoloader files used by Composer. Here are the details on its usage:

When you install a new package or add a new class to your Laravel application, Composer automatically updates its `autoload` files so that your application can load the new code. However, if you manually add a new class or modify the directory structure of your application, Composer may not be able to load the new code. In this case, you need to run the `composer dump-autoload` command.

`composer dump-autoload` regenerates the Composer's autoloader files, including `vendor/autoload.php`. This file maps class names to their corresponding file paths. When you run the command, Composer scans your application's `composer.json` file and rebuilds the autoloader files based on the autoload configuration specified in the file.

In summary, `composer dump-autoload` is a command you should run whenever you add a new class manually, modify the directory structure of your application, or make any other changes that require Composer to regenerate the autoloader files.

```sh
cd ~/madxframework
/usr/local/bin/ea-php82 ~/composer.phar dump-autoload
```
